protocol/packages/currencies/src_by_protocol/osmosis-osmosis-usdc_noble/net_main/lease/impl.rs

```diff
@@ -0,0 +1,382 @@
+use currency::{AnyVisitor, Matcher, MaybeAnyVisitResult, SymbolSlice};
+use sdk::schemars;
+
+use crate::{define_currency, define_symbol};
+
+// Resources:
+// 1. Symbol hashes are computed using the SHA256 Hash Generator https://coding.tools/sha256
+// 2. Currencies that come from Axelar are documented at https://docs.axelar.dev/resources
+// 3. IBC routes from https://github.com/Nolus-Protocol/Wiki/blob/main/testnet-rila/currencies.json
+
+define_symbol! {
+    ATOM {
+        // full ibc route: transfer/channel-0/transfer/channel-0/uatom
+        bank: "ibc/6CDD4663F2F09CD62285E2D45891FC149A3568E316CE3EBBE201A71A78A69388",
+        // full ibc route: transfer/channel-0/uatom
+        dex: "ibc/27394FB092D2ECCD56123C74F36E4C1F926001CEADA9CA97EA622B25F41E5EB2",
+    }
+}
+define_currency!(Atom, ATOM, 6);
+
+define_symbol! {
+    ST_ATOM {
+        // full ibc route: transfer/channel-0/transfer/channel-326/stuatom
+        bank: "ibc/FCFF8B19C61677F3B78E2A5AE3B4A34A8D23858D16905F253B8438B3AFD07FF8",
+        // full ibc route: transfer/channel-326/stuatom
+        dex: "ibc/C140AFD542AE77BD7DCC83F13FDD8C5E5BB8C4929785E6EC2F4C636F98F17901",
+    }
+}
+define_currency!(StAtom, ST_ATOM, 6);
+
+define_symbol! {
+    OSMO {
+        // full ibc route: transfer/channel-0/uosmo
+        bank: "ibc/ED07A3391A112B175915CD8FAF43A2DA8E4790EDE12566649D0C2F97716B8518",
+        dex: "uosmo",
+    }
+}
+define_currency!(Osmo, OSMO, 6);
+
+define_symbol! {
+    ST_OSMO {
```

Comment:
In the main branch, ST_OSMO is present as a lease currency for the main net. However, it is now defined as not supported.